clio-client/src/main/scala/org/broadinstitute/clio/client/commands/MoveWgsUbamCommand.scala

```diff
@@ -0,0 +1,115 @@
+package org.broadinstitute.clio.client.commands
+
+import akka.http.scaladsl.model.HttpResponse
+import com.typesafe.scalalogging.{LazyLogging, Logger}
+import io.circe.Json
+import org.broadinstitute.clio.client.parser.BaseArgs
+import org.broadinstitute.clio.client.util.IoUtil
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.transfer.model.{
+  TransferWgsUbamV1Key,
+  TransferWgsUbamV1Metadata,
+  TransferWgsUbamV1QueryInput
+}
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location}
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+object MoveWgsUbamCommand extends Command with LazyLogging {
+
+  implicit val implicitLogger: Logger = logger
+
+  implicit class FutureErrorMessage[A](future: Future[A]) {
+    def withErrorMsg(message: String)(implicit ec: ExecutionContext,
+                                      logger: Logger): Future[A] = {
+      future andThen {
+        case Success(a) => Future.successful(a)
+        case Failure(ex) =>
+          logger.error(message)
+          Future.failed(new Exception(message, ex))
+      }
+    }
+  }
+
+  override def execute(webClient: ClioWebClient, config: BaseArgs)(
+    implicit ec: ExecutionContext
+  ): Future[HttpResponse] = {
+    for {
+      wgsUbamsResponse <- queryForWgsUbam(webClient, config) withErrorMsg
+        "Could not query the WgsUbam. No files have been moved"
+      firstJson <- getFirstObject(wgsUbamsResponse) withErrorMsg
+        "Could not get only 1 WgsUbam from Clio. No files have been moved"
+      moveWgsUbam <- moveGoogleObject(
```

Comment:
It would be safer to implement 'move' in three steps:

- copy
- update location in ES
- delete original